GOOD-I-DEER/nodes/cosine-similarity-node/calculate-cosine.js

```diff
@@ -70,10 +72,16 @@ module.exports = function (RED) {
         } else {
           result = devideSimilarity(input_vectors, stored_vectors);
         }
-  
-        msg.payload = result;
-        msg.topid = "Result";
-        this.send(msg);  
+
+        if (!result) {
+          msg.payload = "The Vector is not available to calculate.";
+          msg.topic = "Error";
```

Comment:
why msg has 'topic' value?

Reply:
I used this to distinguish correct return value and others. But now I deleted it.